creator-node/test/dbManager.test.js

```diff
@@ -1068,3 +1095,209 @@ describe('Test fetchFilesHashFromDB()', async function () {
     assert.deepEqual(actualResp, expectedResp)
   })
 })
+
+describe('Test fixInconsistentUser()', async function () {
+  const userId = 1
+
+  let server, app
+
+  /** Init server to run DB migrations */
+  before(async function () {
+    const appInfo = await getApp(getLibsMock())
+    server = appInfo.server
+    app = appInfo.app
+  })
+
+  beforeEach(async function () {
+    await destroyUsers()
+  })
+
+  /** Wipe all CNodeUsers + dependent data */
+  after(async function () {
+    await destroyUsers()
+
+    await server.close()
+  })
+
+  it('Confirm no change to healthy users DB state', async function () {
+    const { cnodeUserUUID, walletPublicKey, sessionToken } = await createStarterCNodeUser(userId)
+
+    // Upload some state for user
+    const audiusUserBlockNumber = 10
+    const audiusUserMetadata = { test: 'field1' }
+    const metadataCID = 'QmQMHXPMuey2AT6fPTKnzKQCrRjPS7AbaQdDTM8VXbHC8W'
+    await uploadAudiusUserState({
+      app,
+      userId,
+      sessionToken,
+      metadataObj: audiusUserMetadata,
+      audiusUserBlockNumber
+    })
+    const expectedCNodeUserClock = 2
+
+    // Confirm expected initial state
+    const {
+      cnodeUser: initialCNodeUser,
+      audiusUsers: initialAudiusUsers,
+      tracks: initialTracks,
+      files: initialFiles,
+      clockRecords: initialClockRecords
+    } = await fetchDBStateForWallet(walletPublicKey, models)
+    assertTableEquality(
+      [initialCNodeUser],
+      [{ cnodeUserUUID, walletPublicKey, latestBlockNumber: audiusUserBlockNumber, clock: expectedCNodeUserClock }],
+      ['createdAt', 'updatedAt', 'lastLogin']
+    )
+    assertTableEquality(
+      initialAudiusUsers,
+      [{ cnodeUserUUID, clock: expectedCNodeUserClock, blockchainId: `${userId}`, metadataJSON: audiusUserMetadata, coverArtFileUUID: null, profilePicFileUUID: null }],
+      ['createdAt', 'updatedAt', 'metadataFileUUID']
+    )
+    assertTableEquality(initialTracks, [])
+    assertTableEquality(
+      initialFiles,
+      [{ cnodeUserUUID, trackBlockchainId: null, multihash: metadataCID, sourceFile: null, fileName: null, dirMultihash: null, storagePath: DiskManager.computeFilePath(metadataCID, false), type: "metadata", clock: 1, skipped: false }],
+      ['fileUUID', 'createdAt', 'updatedAt']
+    )
+    assertTableEquality(
+      initialClockRecords,
+      [
+        { cnodeUserUUID, clock: 1, sourceTable: "File" },
+        { cnodeUserUUID, clock: 2, sourceTable: "AudiusUser" }
+      ],
+      ['createdAt', 'updatedAt']
+    )
+
+    // Call fixInconsistentUser()
+    const numRowsUpdated = await DBManager.fixInconsistentUser(cnodeUserUUID)
+    assert.strictEqual(numRowsUpdated, 1)
```

Comment:
shouldn't this be 0?

Reply:
nah bc it always updates it, the function does not actually have conditional logic
kinda useless here, but if value = 0 indicates row doesn't even exist, so i logged it jic
thats why i asserted on all the db table state being unchanged

also appreciate you reading through the test code <3

Reply:
ah i see, it should always be 1 if the cnodeUserUUID is valid because it should always update. this is technically correct i guess

Reply:
exactly